RestaurantReservation.Db/Services/ICustomerService.cs

```diff
@@ -0,0 +1,22 @@
+﻿using RestaurantReservation.Db.Models;
+
+namespace RestaurantReservation.Db.Services
+{
+    public interface ICustomerService
+    {
+        /// <returns>The ID of the created object.</returns>
```

Comment:
The write wait to add a description to a function like the following:
```csharp
        /// <summary>
        /// DESCRIPTON
        /// </summary>
        /// <param name="newEmployee"></param>
        /// <returns></returns>

```

Reply:
And in CustomerService, above CreateAsync, you add  `/// <inheritdoc />` . Which will just inherit the description from the interface

Reply:
![image](https://github.com/Mohammad-Nayef/RestaurantReservation/assets/116375439/bf5e0ad9-d2fe-4198-a0db-5dc37c27ed08)

I see that it's by default inheriting the description of the implemented interface.